src/main/java/uk/ac/ebi/eva/pipeline/parameters/JobParametersNames.java

```diff
@@ -85,6 +85,8 @@
 
     public static final String DB_COLLECTIONS_ANNOTATION_METADATA_NAME = "db.collections.annotation.metadata.name";
```

Comment:
According to previous conversations this should be "db.collections.annotation-metadata.name" (notice the dash instead of dot).